app/controllers/webhook_deliveries_controller.rb

```diff
@@ -0,0 +1,38 @@
+class WebhookDeliveriesController < ApplicationController
+  def index
+    page = params[:page] || 1
+    size = params[:size] || 10
+    client_application_id = params[:client_application_id]
+
+    response = ClientApplicationService.fetch_webhook_deliveries(
+      session[:access_token],
+      client_application_id: client_application_id, page: page, size: size
+    )
+
+    @webhook_deliveries = response.map do |webhook_delivery_data|
+      build_webhook_delivery(webhook_delivery_data)
+    end
+
+    @total_pages = 1 # FIX ME: response.headers["X-Total-Count"]
```

Comment:
Right now we dont any webhook deliveries, let alone pages of them
Hardcoding for now, I'll do a fast follow in a separate clean up pass

Reply:
Pinned to one in this video just to prove that pagination works correctly when communication with Gravity 

https://github.com/user-attachments/assets/8ac913c6-8c26-46bf-a922-715ee6fc50cf